core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -484,6 +484,7 @@ def _convert_proba_to_unified_form(self, y_pred_proba):
         For regression, converts y_pred_proba to a 1 dimensional array of predictions.
         """
         if self.problem_type == REGRESSION:
+            y_pred_proba = np.squeeze(y_pred_proba, axis=-1)
```

Comment:
Why should we add this line?

Reply:
Here ret.as_numpy() will return an array of size (batch X 1), which cannot be handled by _convert_proba_to_unified_form. @sxjscience 
https://github.com/awslabs/autogluon/blob/c521dbf6010bcda31ec3b9f3de37cd51411fb238/vision/src/autogluon/vision/predictor/predictor.py#L503-L504

Reply:
I feel that you can try to change the integration of the vision model instead of touch the abstract_model